core/__tests__/utils/PINCreationValidation.test.ts

```diff
@@ -132,7 +132,7 @@ describe('PIN creation validations', () => {
     const PINValidations = PINCreationValidations(PINWithSeriesOfNumbers, PINRulesWithNoSeriesOfNumbers)
 
     for (const PINValidation of PINValidations) {
-      if (PINValidation.errorName === 'SeriesOfThreeNumbersValidation') {
+      if (PINValidation.errorName === 'NoSeriesOfNumbersValidation') {
```

Comment:
Not for this PR but if you're working on this feature in the future this may be well suited for an `enum`.